SECURITY.md

```diff
@@ -18,6 +18,14 @@ As Git for Windows bundles more than just Git (such as Bash, OpenSSL, OpenSSH, G
 
 Every Git for Windows version is tagged using a name that starts with the Git version on which it is based, with the suffix `.windows.<patchlevel>` appended. For example, Git for Windows v2.17.1' source code is tagged as [`v2.17.1.windows.1`](https://github.com/git-for-windows/git/releases/tag/v2.17.1.windows.1) (the patch level is always at least 1, given that Git for Windows always has patches on top of Git). Likewise, Git for Windows v2.17.1(2)' source code is tagged as [`v2.17.1.windows.2`](https://github.com/git-for-windows/git/releases/tag/v2.17.1.windows.2).
 
+## Release Candidates (rc) and Snapshot versions ('nightlies')
+
+Git for Windows also releases versions that reflect the upstream release candidates, which contain the `-rc<n>` suffix to the regular git version, and before the 'windows' suffix. It should be noted that these rc versions sort after their formal release, so appear to be newer to the updater software.
+
+[All releases](https://github.com/git-for-windows/git/releases/) are listed via a link at the footer of the [Git for Windows](https://gitforwindows.org/) home page.
+
+Git for Windows also provides snapshots (these are not releases) of the progressing upstream development via the shears/* and vs/* branches and the [Snapshots](https://wingit.blob.core.windows.net/files/index.html) page. These snapshots contain the Windows specific patches via automated continuous integration. Link also at the footer of the [Git for Windows](https://gitforwindows.org/) home page.
```

Comment:
Actually, there are no snapshots for `shears/*`. There are only snapshots for _our_ `master`.

At some stage in the future, I would like to get an Azure Pipeline going that automatically builds portable Git and MinGit versions whenever `shears/*` was updated _and_ passes the test suite. This will require a bit of effort, and I do not have the time right now.